packages/api/tests/endpoints/test-execution-status.js

```diff
@@ -38,7 +38,6 @@ const {
 } = require('../../lib/testUtils');
 
 process.env.AccessTokensTable = randomString();
-process.env.ExecutionsTable = randomString();
```

Comment:
I stumbled on this in my Granule PR: https://github.com/nasa/cumulus/pull/3263/files#diff-d05cfbcd00b8fd1725d2daf3802ba636613c5b2910e224ad2e9b7e6f2b3e9104L41